src/main/java/org/folio/inventory/common/dao/EventIdStorageDaoImpl.java

```diff
@@ -0,0 +1,47 @@
+package org.folio.inventory.common.dao;
+
+import io.vertx.core.Future;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import io.vertx.sqlclient.Tuple;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.inventory.domain.relationship.EventTable;
+import org.folio.inventory.domain.relationship.EventToEntity;
+
+import java.util.UUID;
+
+public class EventIdStorageDaoImpl implements EventIdStorageDao {
+  private static final Logger LOGGER = LogManager.getLogger(EventIdStorageDaoImpl.class);
+
+  private static final String INSERT_FUNCTION = "INSERT INTO {schemaName}.{tableName} VALUES ($1::uuid, current_timestamp) RETURNING *;";
```

Comment:
Due to the nature of this table, it can get intense(high cpu) when allowing the database to set the "current date". Under high load it is better to generate the date from the app tier and just send the data to the database.

Reply:
@okolawole-ebsco  As far as I remember, in most places, we left this datetime-generation to the DB side. 
This is good point for the future (maybe to move this generation to the Java-code, not functions/trigger on the DB-side...?)
So yes, I improved it here - moved this generation to the codebase.
Could you please double-check it?